### ISSUE TITLE
Unblock lists.sourceforge.net

### ISSUE THREAD

**Issue by hickford** [gorhill/uBlock#938]
Unless there's any risk from reading mailing lists, would you consider unblocking https://lists.sourceforge.net/ ? 

https://github.com/gorhill/uBlock/wiki/Badware-risks

**Comment by gorhill** [comment]
When I click `https://lists.sourceforge.net/` I am being redirected to `http://sourceforge.net/`.

**Comment by hickford** [comment]
https://lists.sourceforge.net/lists/listinfo/ssllabs-discuss was the precise website I was trying to read

**Comment by publicarray** [comment]
Can we keep these discussion in one issue? related: #561 #871 #937.

You can simply unblock the subdomain with a button press or filter:
`no-strict-blocking: lists.sourceforge.net true`

**Comment by gorhill** [comment]
Duplicate of #937.
